spirv_msl.cpp

```diff
@@ -2599,7 +2722,7 @@ string CompilerMSL::entry_point_args(bool append_comma)
 			{
 				if (!ep_args.empty())
 					ep_args += ", ";
-				BuiltIn bi_type = meta[var_id].decoration.builtin_type;
+				BuiltIn bi_type = (BuiltIn)get_decoration(var_id, DecorationBuiltIn);
```

Comment:
This is a bit awkward with the casting back to an enum which meta[] does not need.

Reply:
Reverted.

Reply:
Removed them.